lib/presentation/scenes/list_movie/bloc/list_movie_cubit.dart

```diff
@@ -2,41 +2,62 @@ import 'package:equatable/equatable.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:the_movie_database/api_service/core/authentication/list-movies/list_movies_params.dart';
 import 'package:the_movie_database/api_service/core/authentication/list_movies_api_input.dart';
-import 'package:the_movie_database/api_service/core_api_service.dart';
-import 'package:the_movie_database/model/list_response_object/list_response_object.dart';
+import 'package:the_movie_database/domain/usecases/get_list_movies_usecase.dart';
+import 'package:the_movie_database/model/movie_response_object/movie_response_object.dart';
+import 'package:the_movie_database/utils/utils.dart';
 
 part 'list_movie_state.dart';
 
 class ListMovieCubit extends Cubit<ListMovieState> {
-  ListMovieCubit() : super(ListMovieState.initial());
+  ListMovieCubit(this._getListMoviesUseCase) : super(ListMovieState.initial());
+  final GetListMoviesUseCase _getListMoviesUseCase;
 
-  Future<void> getListPopular() async {
+  Future<void> getListMovies({int page = 1, String title = ""}) async {
     emit(
       state.copyWith(
         isLoading: true,
         isEnd: false,
+        page: page,
       ),
     );
+    String url = '';
+    print('titletitle $title');
+    switch (title) {
```

Comment:
dart 3.1 trở lên thì dùng cái này đi
https://dart.dev/language/patterns#switch-statements-and-expressions